File: Question5/Question5.py
# Your code goes here
vowels = ['a', 'e', 'i', 'o', 'u']
def find_vowels(str):
    string = str.lower()
    vowel_count = 0
    vowels_found = []
    for letter in string:
        if letter in vowels:
            vowels_found.append(letter)
            vowel_count += 1

    return vowel_count, vowels_found

# Test Cases

print(find_vowels("Hello World!"))
print(find_vowels("Uppercase Example"))

File: Question5/Question5.js
/// Your code goes here

function find_vowels(str) {
const vowels = ["a", "e", "i", "o", "u"]
let vowel_count = 0
let vowel_array = []
str = str.toLowerCase();


// loop over each letter in the string and add any vowel to vowel_array & vowel_count ++
for (let i = 0; i < str.length; i++) {
    const new_i = str[i];
}

if (vowels.includes(new_i)) {
    vowel_count++;
}

if (!vowel_array.includes(new_i)) {
    find_vowels.push(new_i)
}
}

/// Test Cases

console.log(vowel_array("momentum"))
console.log(vowel_count)
